### DISCUSSION TITLE
Is there a way to get Markdown text using frontend API?

### DISCUSSION THREAD

**Discussion by asharca** [TriliumNext/Trilium#6014] [Ideas]
I want to sync my Trilium notes with GitHub. I've created a button that should export the note's Markdown content and push it to GitHub,
When I try to get the note content using:
```javascript
const note = await api.getActiveContextNote();
const content = note.getContent();
```
it returns HTML instead of Markdown.
is there a way to access the original Markdown content through the frontend API? I've looked through the documentation but haven't found a clear answer.

**Comment by Nriver** [comment]
Well, the default note type are acutally HTML data, there is no `original markdown`. You can find the source code of your note content in the three dots on the upper right corner.

If you want the markdown, you need to change the note type to markdown.
